### ISSUE TITLE
Just a question about suffix -sori

### ISSUE THREAD

**Issue by lewisl** [eueung/hugo-casper-two#23]
For partial layouts that end with suffix -sori what is different and is the use of -sori version or without the suffix determined by choice of "casper" or "caspertwo"?

Thanks.

**Comment by dixonge** [comment]
@lewisl Yes, casper vs caspertwo determines that - you can see a lot of if-then statements in the layout files, mainly 'if casper do this, else do that'
